src/api/components/PostList.tsx

```diff
@@ -0,0 +1,23 @@
+import { PostInfo } from './PostInfo';
+
+type Props = {
+  posts: Post[];
+};
+
+export const PostList: React.FC<Props> = ({ posts = [] }) => (
+  <ul>
+    {posts.map(post => (
+      <>
+        <li key={post.id}>
+          {post.user && (
+            <PostInfo
+              post={post}
+              user={post.user}
+              comments={post.comments}
```

Comment:
you pass user and comments twice
```suggestion
              post={post}
```